benchmark/http2/headers.js

```diff
@@ -6,7 +6,8 @@ const PORT = common.PORT;
 const bench = common.createBenchmark(main, {
   n: [1e3],
   nheaders: [0, 10, 100, 1000],
-}, { flags: ['--no-warnings'] });
+  benchmarker: ['h2load']
+}, { flags: ['--no-warnings', '--expose-http2'] });
```

Comment:
Any specific reason `--expose-http2` is back in? Is it just so compare can be run against versions where that was required?

Reply:
Yep. It's for benchmark comparisons against v8.x